.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "xdr/next"]
 	path = xdr/next
-	url = https://github.com/stellar/stellar-xdr
-	branch = next
+	url = https://github.com/jayz22/stellar-xdr
```

Comment:
I've approved the associated change on stellar/stellar-xdr so you can switch this reference back

Reply:
I have switched back the reference.